### ISSUE TITLE
[Windows 7] FileDialog system icons shown as dots

### ISSUE THREAD

**Issue by baytuni** [ColinDuquesnoy/QDarkStyleSheet#51]
When I use qfiledialog icons like up,forward, back etc appear as dots

![image](https://cloud.githubusercontent.com/assets/23097617/24739333/bb353dd8-1a9b-11e7-966a-679ca7f1c6ed.png)

**Comment by ColinDuquesnoy** [comment]
Thank you for reporting. Unfortunately I am afraid I cannot help. I don't use Windows 7 anymore and there is no issue on Windows 10.

If you (or anyone else) find a solution submit a PR and I'll merge it.

**Comment by baytuni** [comment]
Hey,
I am sorry I stirred things up a bit. It turns out it's due to lineedit padding. I commented the padding line for other purposes but it affects the icons in the file dialog.  I solved the problem by adjusting the padding for all the elements appearing on the qfiledialog. Whoever interested might check the file. I don't wanna branch the code cause I changed the colors too so I share it as a text file. 
[style.txt](https://github.com/ColinDuquesnoy/QDarkStyleSheet/files/912487/style.txt)
